src/jit/inlinepolicy.h

```diff
@@ -216,13 +216,45 @@ class DiscretionaryPolicy : public LegacyPolicy
     const char* GetName() const override { return "DiscretionaryPolicy"; }
 
 private:
+    
+    void ComputeOpcodeBin(OPCODE opcode);
+    enum { MAX_ARGS = 4 };
```

Comment:
Ah, yes, I was foolishly thinking that MAX_ARGS of 4 would mean we would include methods with 4 args, but really that limits to 3 args, since the 0th item in this count is not an arg ;-)